doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -458,6 +458,9 @@ Bluetooth Fast Pair samples
     * The location of the sample configuration.
       It has been moved from the root sample directory to the dedicated folder (:file:`locator_tag/configuration`).
     * The ``fp_adv`` module to use the trigger requests for the Fast Pair advertising state instead of setting the Fast Pair advertising mode directly.
+    * Sample configuration to use a separate workqueue for connection TX notify processing (:kconfig:option:`CONFIG_BT_CONN_TX_NOTIFY_WQ`).
```

Comment:
Consider moving it to the common changelog section and combine the change with the Input Device sample update:

https://github.com/nrfconnect/sdk-nrf/pull/18088